index.html

```diff
@@ -166,130 +166,95 @@ <h2 class="section-heading text-uppercase">Technologies Used</h2>
                     <h3 class="section-subheading text-muted">Cutting Edge Technology is what we specialize in. </h3>
                 </div>
                 <div class="row">
-                    <div class="col-lg-4 col-sm-6">
+                    <div class="col-lg-4 col-sm-6 mb-4">
                         <!-- Portfolio item 1-->
-                        <div class="portfolio-item">
-                            <a class="portfolio-link" data-bs-toggle="modal" href="#portfolioModal1">
-                                <div class="portfolio-hover">
-                                    <div class="portfolio-hover-content"><i class="fas fa-plus fa-3x"></i></div>
-                                </div>
-                                <img class="img-fluid" src="img/MERN_blog.png" height="400" width="500" alt="..."/>
-                            </a>
-                            <div class="portfolio-caption">
-                                <div class="portfolio-caption-heading">MERN Stack</div>
-                               
+                        <div class="box">
+                            <h3>MERN Stack</h3>
+                            <img src="img/MERN_blog.png" alt="..." height="150" width="200" class="img-fluid"/>

```

Comment:
Don't hard code the height and width of the same image time. You can use width and remove the height element here. The height will be dynamically allocated according to the image size.